tests/py/test_billing_payday.py

```diff
@@ -501,8 +432,8 @@ def test_process_draws(self):
         with self.db.get_cursor() as cursor:
             payday.prepare(cursor)
             payday.process_payment_instructions(cursor)
-            payday.transfer_takes(cursor, payday.ts_start)
-            payday.process_draws(cursor)
+            payday.process_remainder(cursor)
+            payday.process_takes(cursor, payday.ts_start)
```

Comment:
Doesn't make a difference functionally since there are no `takes` in the picture, but the order these should execute in is `process_takes` followed by `process_remainder`, not the opposite